### ISSUE TITLE
Prevent orphaned contact cards

### ISSUE THREAD

**Issue by MizukiTemma** [digitalfabrik/integreat-cms#3260]
### Motivation

#3169 introduces a functionality to embed a contact as a card into contents (pages, events, locations). It is currently possible to archive/delete a contact object that is embedded in a content. After archiving/deleting an embedded contact object, its contact card remains in the content. This will be chaoric in future.

### Proposed Solution

Either 
-  Prevent archiving/deleting contact objects that are embedded somewhere (like objects of other models)
or
- <s>Delete embedded contact cards when their contact objects is archived/deleted (with warning and confirmation button)</s>

### Alternatives

Ignore this potential problem

### Additional Context

Wait for #3169

**Comment by charludo** [comment]
Ohh, good catch!!

**Comment by JoeyStk** [comment]
@osmers We would lean towards the first proposed solution. Do you agree with us? :)

**Comment by osmers** [comment]
Yeah, I agree - it's the same behaviour we already have with live content and POIs used in events for example, right? Would make sense then to have a similar list e.g.

This contact cannot be archived because it is used by the following page/event/POI
